### ISSUE TITLE
Broken Missing files - Bug updating videos listing

### ISSUE THREAD

**Issue by gbrn** [WWBN/AVideo#9031]
@DanielnetoDotCom Danny, in some videos that are presenting Broken Missing files. I will upload the video file again to correct the problem. But the system keeps saying Broken Missing files even with the correct files in the directory. Only when I enable the video, the system finds the modified file.
I don't understand how the file checking cycle works, whether it is from time to time or every time the MVIDEO video listing is opened.

Check:

https://github.com/WWBN/AVideo/assets/37880073/07668c93-07bd-4f14-8500-3b49207428f2

**Comment by DanielnetoDotCom** [comment]
When you say "Only when I enable the video", do you mean make them active?

**Comment by gbrn** [comment]
Yes, when I activate the video it works. But before that the error keeps "Broken Missing files". This causes a false negative. And in fact I don't know if it's correct if I don't activate the video. The video above shows this happening.

**Comment by DanielnetoDotCom** [comment]
I just sent an update that may give a better log when your file turns into a "Broken Missing file"

please update, reproduce the error, and share the log.

**Comment by gbrn** [comment]
@DanielnetoDotCom Before the pull, I did a test and a video that was in the "Broken Missing file" status, I uploaded the poster and it went from the "Broken Missing file" status to "Active".
I noticed that the system was not updating the status automatically, only after some command for the video, such as manually activating a video or updating an image.

I will run new tests after the PULL and notify you and provide feedback.

**Comment by gbrn** [comment]
Check out this error (which happened before the last PULL):
Video with status "Broken Missing files":
![image](https://github.com/WWBN/AVideo/assets/37880073/732f40ce-9c3d-448e-b4ff-5aae2ce33d44)

I try to access the file directly.... it shows the message "Not Found"
https://cnetv.com.br/videos/video_220911225633_va98d/video_220911225633_va98d_HD.mp4
![image](https://github.com/WWBN/AVideo/assets/37880073/f2f24b28-3128-460e-878c-ccecc5c287ae)

The file exists on the server, but is only 285kb:
![image](https://github.com/WWBN/AVideo/assets/37880073/83f62534-0d09-48ba-9c2d-1c06f8885250)

It is important to note that I did not change anything in this file and it was corrupted by some script.

**Comment by gbrn** [comment]
@DanielnetoDotCom Danny... this video here is also in the status of "Broken Missing files".
But different from the previous video reported. I was able to locate the file that was transferred from the local server to the storage server. Although the system does not find the file.

LOCAL: https://cnetv.com.br/videos/_YPTuniqid_609b57e85c3fe3.70429701/_YPTuniqid_609b57e85c3fe3.70429701_720.mp4

STORAGE: https://storage01.cnetv.com.br/videos/_YPTuniqid_609b57e85c3fe3.70429701/_YPTuniqid_609b57e85c3fe3.70429701_720.mp4

![image](https://github.com/WWBN/AVideo/assets/37880073/9a4f3bca-fe56-4375-a303-71c0f0fd3e5b)

**Comment by gbrn** [comment]
@DanielnetoDotCom I verified that the content of the MP4 file is HTML... would this be the DUMP file?
I think this file was converted when transferring from the LOCAL server to the STORAGE server.

![image](https://github.com/WWBN/AVideo/assets/37880073/53c5e541-0a08-467b-b1ab-e577900c795d)

**Comment by DanielnetoDotCom** [comment]
I am not sure what is happening, 

why are you downloading files from your storage to the local?

what is the modification time?

what happen in your log in the modification time?
